generator/discovery/update_discovery_doc.sh

```diff
@@ -69,24 +78,60 @@ readonly REVISION
 io::run git checkout -B update_compute_discovery_circa_"${REVISION}"
 
 io::log_h2 "Adding updated Discovery JSON ${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}"
-git add "${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}"
+git commit -m"chore(compute): update discovery doc circa ${REVISION}" \
+  "${PROJECT_ROOT}/${COMPUTE_DISCOVERY_JSON_RELATIVE_PATH}"
 
 io::log_h2 "Running generate-libraries with UPDATED_DISCOVERY_DOCUMENT=compute"
 UPDATED_DISCOVERY_DOCUMENT=compute ci/cloudbuild/build.sh -t generate-libraries-pr
 
+io::log_h2 "Adding new compute internal protos"
+git add "${PROJECT_ROOT}/protos/google/cloud/compute/v1/internal"
+
 NEW_FILES=$(git ls-files --others --exclude-standard protos/)
 if [[ -n "${NEW_FILES}" ]]; then
+  git add protos/google/cloud/compute
+
+  io::log_h2 "Formatting generated protos"
+  git ls-files -z -- '*.proto' |
+    xargs -P "$(nproc)" -n 50 -0 clang-format -i
+  git ls-files -z -- '*.proto' |
+    xargs -r -P "$(nproc)" -n 50 -0 sed -i 's/[[:blank:]]\+$//'
+
   io::log_red "New resources defined in Discovery Document created new protos:"
   echo "${NEW_FILES}"
   mapfile -t proto_array < <(echo "${NEW_FILES}")
-  io::log_red "Add rest_services definitions to the generator_config.textproto and re-run this script."
+  io::log_red "Adding rest_services definitions to the generator_config.textproto."
   for i in "${proto_array[@]}"; do
+    echo "${i}"
     print_service_textproto "${i}"
   done
-  git add protos
-  io::log_yellow "Adding new directories to google/cloud/compute/service_dirs.cmake"
+fi
+
+io::log_h2 "Running generate-libraries with updated generator_config.textproto"
+ci/cloudbuild/build.sh -t generate-libraries-pr || true
+
+if [[ -n "${NEW_FILES}" ]]; then
+  io::log_h2 "Adding new compute generated service code"
+  git add google/cloud/compute
+
+  io::log_h2 "Formatting generated code"
+  git ls-files -z -- '*.h' '*.cc' |
+    xargs -P "$(nproc)" -n 50 -0 clang-format -i
+  git ls-files -z -- '*.h' '*.cc' |
+    xargs -r -P "$(nproc)" -n 50 -0 sed -i 's/[[:blank:]]\+$//'
```

Comment:
nit: we are not pressed for time or anything, but we could scope these to `google/cloud/compute`